### ISSUE TITLE
CodePeak 2025 : Medical Insurance Cost  Prediction

### ISSUE THREAD

**Issue by vanshika230** [prathimacode-hub/ML-ProjectKart#569]
Define You:

- [x] CodePeak 2025 Participant 
- [ ] Contributor

## I'd like to make a Medical Insurance Cost Prediction Model using Linear Regression.

-  It  uses Age, Sex, bmi, children, smoker and region as Independent features 
-  The dependent feature is the Medical Insurance charges

## Workflow :-

1️⃣  Data collection and analysis
2️⃣ Data Preprocessing
3️⃣  Train_test_spilt
4️⃣  Applying Linear Regression Model
5️⃣  Finding accuracy and evaluating model 

@prathimacode-hub I'd like to work on this.

**Comment by prathimacode-hub** [comment]
Yes, you can proceed but use multiple algorithms and provide proper comparison of accuracies at end with the conclusion. @vanshika230

**Comment by tanvideshmukh29** [comment]
@prathimacode-hub
I would love to contribute mam as a part of hacktoberfest2021.
Can you assign this to me?

**Comment by vanshika230** [comment]
Ma'am I'll upload the work by tomorrow, I had exams till today so I couldn't do it. Apologies for the late work and failure to update🙏

**Comment by prathimacode-hub** [comment]
By when are you expected to do? Fest is about to end? @vanshika230

**Comment by surya-moorthy** [comment]
I would like to work on this issue can you assigned me ?
